spec/services/form1010cg/auditor_spec.rb

```diff
@@ -286,4 +286,25 @@
       end
     end
   end
+
+  describe '#filtering keys' do
+    let(:message) { 'my message' }
+    let(:context) do
+      {
+        veteran_name: 'My name'
+      }
+    end
+
+    let(:expected_context) do
+      {
+        veteran_name: ActiveSupport::ParameterFilter::FILTERED
+      }
+    end
+
+    it 'calls :record_attachments_delivered' do
+      expect(Rails.logger).to receive(:info).with("[#{described_class::LOGGER_PREFIX}] #{message}", **expected_context)
+      
+      described_class.new.send(:log, message, context)
+    end
```

Comment:
🤔 not sure what using `let`s offers here? might read clearer in-line?
```suggestion
  describe '#filtering keys' do
    it 'calls :record_attachments_delivered' do
      message = 'my message'
      context = { veteran_name: 'My name' }
      expected_context = { veteran_name: ActiveSupport::ParameterFilter::FILTERED }
      
      expect(Rails.logger).to receive(:info).with("[#{described_class::LOGGER_PREFIX}] #{message}", **expected_context)
      
      described_class.new.send(:log, message, context)
    end
```